src/components/MoviesList/MoviesList.tsx

```diff
@@ -4,12 +4,19 @@ import { MovieCard } from '../MovieCard';
 
 interface Props {
   movies: Movie[];
+  filter: string;
 }
 
-export const MoviesList: React.FC<Props> = ({ movies }) => (
-  <div className="movies">
-    {movies.map(movie => (
-      <MovieCard key={movie.imdbId} movie={movie} />
-    ))}
-  </div>
-);
+export const MoviesList: React.FC<Props> = ({ movies, filter }) => {
+  return (
+    <div className="movies">
+      {movies.filter(v => v.title.toLowerCase()
+        .includes(filter.toLowerCase().trimStart().trimEnd())
+        || v.description.toLowerCase()
+          .includes(filter.toLowerCase().trimStart().trimEnd()))
+        .map(movie => (
+          <MovieCard key={movie.imdbId} movie={movie} />
+        ))}
```

Comment:
this logic should not be included in JSX - always when possible we want to have our rendering and business logic separate, please move it up to `App.tsx`, and separate it into another function